### ISSUE TITLE
eslint reported issues issues clean up

### ISSUE THREAD

**Issue by TheObstinatePanda** [Jamesllllllllll/Daily-Dashboard#42]
While we work to clear the main issue, I'm going to try to track these eslint issues which seems to have piled up on us. 

```
[eslint]
src\App.js
  Line 7:10:  'ThemeProvider' is defined but never used  no-unused-vars
  Line 7:25:  'createTheme' is defined but never used    no-unused-vars

src\components\QuoteOrFact\subComponents\Quote.js
  Line 10:19:  'setLoading' is assigned a value but never used
                                                                no-unused-vars
  Line 86:9:   The 'fetchQuote' function makes the dependencies of useEffect Hook (at line 104) change on every render. Move it inside the useEffect callback. Alternatively, wrap the definition of 'fetchQuote' in its own useCallback() Hook  react-hooks/exhaustive-deps

src\components\ShowHistory\ShowHistory.js
  Line 21:46:  Function declared in a loop contains unsafe references to variable(s) 'existingDay'  no-loop-func
  Line 21:53:  Array.prototype.map() expects a value to be returned at the end of arrow function    array-callback-return
  Line 45:44:  Function declared in a loop contains unsafe references to variable(s) 'existingDay'  no-loop-func
  Line 45:51:  Array.prototype.map() expects a value to be returned at the end of arrow function    array-callback-return

src\components\WeatherForm\WeatherForm.jsx
  Line 6:3:  'updateWeather' is defined but never used  no-unused-vars

src\features\to-do-list\NewTask.js
  Line 3:17:  'useState' is defined but never used  no-unused-vars

src\index.js
  Line 13:10:  'CssBaseline' is defined but never used  no-unused-vars

Search for the keywords to learn more about each warning.
To ignore, add // eslint-disable-next-line to the line before.

WARNING in [eslint] 
src\App.js
  Line 7:10:  'ThemeProvider' is defined but never used  no-unused-vars
  Line 7:25:  'createTheme' is defined but never used    no-unused-vars

src\components\QuoteOrFact\subComponents\Quote.js
  Line 10:19:  'setLoading' is assigned a value but never used
                                                                no-unused-vars
  Line 86:9:   The 'fetchQuote' function makes the dependencies of useEffect Hook (at line 104) change on every render. Move it inside the useEffect callback. Alternatively, wrap the definition of 'fetchQuote' in its own useCallback() Hook  react-hooks/exhaustive-deps

src\components\ShowHistory\ShowHistory.js
  Line 21:53:  Array.prototype.map() expects a value to be returned at the end of arrow function    array-callback-return
  Line 45:44:  Function declared in a loop contains unsafe references to variable(s) 'existingDay'  no-loop-func
  Line 45:51:  Array.prototype.map() expects a value to be returned at the end of arrow function    array-callback-return

src\components\WeatherForm\WeatherForm.jsx
  Line 6:3:  'updateWeather' is defined but never used  no-unused-vars

src\features\to-do-list\NewTask.js
  Line 3:17:  'useState' is defined but never used  no-unused-vars

Compiled with warnings.

[eslint]
src\components\QuoteOrFact\subComponents\Quote.js
  Line 10:19:  'setLoading' is assigned a value but never used
                                                                no-unused-vars
  Line 86:9:   The 'fetchQuote' function makes the dependencies of useEffect Hook (at line 104) change on every render. Move it inside the useEffect callback. Alternatively, wrap the definition of 'fetchQuote' in its own useCallback() Hook  react-hooks/exhaustive-deps

Search for the keywords to learn more about each warning.
To ignore, add // eslint-disable-next-line to the line before.

WARNING in [eslint]
src\components\QuoteOrFact\subComponents\Quote.js
  Line 10:19:  'setLoading' is assigned a value but never used
                                                                no-unused-vars
  Line 86:9:   The 'fetchQuote' function makes the dependencies of useEffect Hook (at line 104) change on every render. Move it inside the useEffect callback. Alternatively, wrap the definition of 'fetchQuote' in its own useCallback() Hook  react-hooks/exhaustive-deps
```

**Comment by TheObstinatePanda** [comment]
a branch has been made for this - eslint-issue-patching

**Comment by klunkyfungus** [comment]
I can easily fix this issue:

src\features\to-do-list\NewTask.js
  Line 3:17:  'useState' is defined but never used  no-unused-vars

Before I do so i should switch to the eslint-issue-patching branch and make the changes there?

**Comment by klunkyfungus** [comment]
Nevermind, i see on PR 43 that you fixed these already

**Comment by Jamesllllllllll** [comment]
Closed with PR #43
